docs/standard/serialization/system-text-json-how-to.md

```diff
@@ -698,6 +802,118 @@ With this option, the `Summary` property of the `WeatherForecastWithDefault` obj
+
+:::code language="csharp" source="snippets/system-text-json-how-to-5-0/csharp/QuotedNumbers.cs" highlight="27,37":::
+
+When you use `System.Text.Json` indirectly through ASP.NET Core, quoted numbers are allowed because ASP.NET Core specifies [web default options](xref:System.Text.Json.JsonSerializerDefaults.Web).
+
+::: zone-end
+
+::: zone pivot="dotnet-core-3-1"
+<xref:System.Text.Json> 3.1 doesn't support serializing or deserializing numbers in quotes.
+::: zone-end
+
+## Immutable types and Records
+
+::: zone pivot="dotnet-5-0"
+`System.Text.Json` can use a parameterized constructor, which makes it possible to deserialize an immutable class or struct. For a class, if the only constructor is a parameterized one, that constructor will be used. For a struct, or a class with multiple constructors, specify the one to use by applying the [[JsonConstructor]](xref:System.Text.Json.Serialization.JsonConstructorAttribute.%23ctor%2A) attribute, as shown in the following example:
+
+:::code language="csharp" source="snippets/system-text-json-how-to-5-0/csharp/ImmutableTypes.cs" highlight="13":::
+
+Records in C# 9 are also supported, as shown in the following example:
+
+:::code language="csharp" source="snippets/system-text-json-how-to-5-0/csharp/Records.cs":::
+::: zone-end
+
+::: zone pivot="dotnet-core-3-1"
+`JsonConstructorAttribute` and C# 9 Record support aren't available in `System.Text.Json` 3.1.
+::: zone-end
+
+## Copy JsonSerializerOptions
+
+::: zone pivot="dotnet-5-0"
+There is a <xref:System.Text.Json.JsonSerializerOptions> constructor that lets you create a new instance with the same options as an existing instance, as shown in the following example:
+
+:::code language="csharp" source="snippets/system-text-json-how-to-5-0/csharp/CopyOptions.cs" highlight="29":::
+::: zone-end
+
+::: zone pivot="dotnet-core-3-1"
+A constructor that takes an existing instance is not available in `System.Text.Json` 3.1.
+::: zone-end
+
+## Web defaults for JsonSerializerOptions
+
+::: zone pivot="dotnet-5-0"
+Here are the options that have different defaults for web apps:
+
+* <xref:System.Text.Json.JsonSerializerOptions.PropertyNameCaseInsensitive%2A> = `true`
+* <xref:System.Text.Json.JsonNamingPolicy> = <xref:System.Text.Json.JsonNamingPolicy.CamelCase>
+* <xref:System.Text.Json.JsonSerializerOptions.NumberHandling%2A> = <xref:System.Text.Json.Serialization.JsonNumberHandling.AllowReadingFromString>
+
+There's a <xref:System.Text.Json.JsonSerializerOptions> constructor that lets you create a new instance with [the default options that ASP.NET Core uses for web apps](xref:System.Text.Json.JsonSerializerDefaults.Web), as shown in the following example:
```

Comment:
Perhaps we should generalize this section to be about the constructor, and the mention web defaults as one possilbe param. This should scale well if we have more defaults later on.

Reply:
> generalize this section to be about the constructor, and then mention web defaults as one possible param. This should scale well if we have more defaults later on.

The emphasis in the conceptual docs is covering scenarios rather than APIs -- for constructor documentation we have the API ref, and for the scenario of needing to apply web defaults we have this section of a conceptual doc, which explains what the defaults are and how to apply them. An alternative is  to just note what the defaults are (since that doesn't show up in the API ref) and link to the constructor API ref, and omit the code example.